src/content/en/fundamentals/performance/rail.md

```diff
@@ -78,18 +83,24 @@ For actions that take longer than 500ms to complete, always provide feedback.
 
 Success: Respond to user's touchmoves and scrolling in under 16ms.
 
-## Animation: render frames every 16ms
+## Animation: execute animation code in under 10ms
```

Comment:
I prefer the "produce a frame" phrasing, but that may be less understandable to developers. My concern with this is that folks will think "animation code" just means rAF callbacks.